stock_picking_report_product_sticker/report/report_deliveryslip.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template
+        id="report_delivery_sticker_document"
+        inherit_id="stock.report_delivery_document"
+    >
+        <xpath expr="//div[hasclass('page')]" position="inside">
+            <div
+                t-if="o.show_product_stickers"
+                name="delivery_sticker_section"
+                class="mt-5"
+            >
+                <t
+                    t-call="stock_picking_report_product_sticker.stock_report_delivery_stickers"
+                />
+            </div>
+        </xpath>
+    </template>
+
+    <template id="stock_report_delivery_stickers">
+        <t t-foreach="o.sticker_ids" t-as="sticker">
+            <div class="col-2 d-inline-block text-center align-top p-0 mt-3 mr-1">
+                <img
+                    t-att-src="image_data_uri(sticker.image_128)"
+                    t-att-alt="sticker.name"
+                    class="m-auto d-block"
+                />
+                <span
+                    t-if="sticker.show_sticker_note and sticker.note"
+                    t-raw="sticker.get_html_sanitized_note()"
```

Comment:
There must be something wrong in the way you're escaping the HTML. See:

![imagen](https://user-images.githubusercontent.com/973709/224025137-58ade9b0-2240-4bc4-93b5-c50b8c244aaf.png)

Reply:
This must be addressed in the Product Sticker module.

Reply:
This way doesn't allow to insert breaks on the note: 
Sticker note: 
```
<text 1>
text 2
```

Will be displayed as shown with `t-out`: 
```
<text 1> text 2
```